tools/cdep/commands/update.go

```diff
@@ -72,8 +77,23 @@ var UpdateCmd = &cobra.Command{
 			return err
 		}
 
+		region := "eu-west-1"
+		awsSession, err := session.NewSessionWithOptions(session.Options{
+			Profile: "root",
+			Config: aws.Config{
+				Region:      &region,
```

Comment:
can use `ptr.String` for this one :) avoids having to set the region as a separate variable